src/Components/Recommendations/BookRec.js

```diff
@@ -0,0 +1,244 @@
+/*
+    ============ TODO =============
+    - Get current book's tags
+    - query airtable for a list of similar books
+    - store the list somewhere
+
+    Prioritization:
+      primary -> age-range, grade-range, race/ethnicity, and identity
+      secondary -> author,  genre, theme/lessons
+
+    Filter:
+      must make sure that the same book is not shown
+
+    How to make sure prioritization is in the correct range:
+      - priorisation: books that have the most correct matches to the top tier.
+      - top tier: age_range, grade_range, race/ethnicity
+      - if books have same number of correct matches,
+        prioritize age_range > grade_range > race_ethnicity
+
+    Challenges:
+      - Tweak filter that can work with array of data
+      - Tweak filter to work with range of values
+
+    If airtable filter doesn't work:
+      fetch all books from airtable
+      apply filters and increment the priority counter for each book accordingly
+      sort based on priority and display the top n books
+
+*/
+
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import BookCards from './BookRecCards';
+
+// airtable configuration
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
+export default function RecCardsDisplay({
+  minAge, maxAge, minGrade, maxGrade, raceEthnicity,
+}) {
+  const [book, setBook] = useState([]);
+  const gradeList = ['0 to Pre-K', 'Kindergarten', '1st', '2nd', '3rd', '4th', '5th', '6th', '7th', '8th', '9th', '10th', '11th', '12th'];
```

Comment:
This constant can be moved out of the function!

Reply:
Is it better to have a global constant or can we just keep it in the function, because I don't think anything else will have to use this constant.

Reply:
There are a lot of components that use this gradeList, so maybe a better final implementation is to add a file for commonly used constants and import it where needed! Can you make a 'Constants' file w/in the src folder and import it here? Then we can add onto it and import where needed!